### ISSUE TITLE
PageHeader component: Tooltip on hover on the long page title in PageHeader component

### ISSUE THREAD

**Issue by Sumeetrana-innovaccer** [innovaccer/design-system#2430]
## Summary

On the PageHeader component, currently if the title is very long it got truncated. But on hover there is no tooltip coming. 

### Basic example

When there is a very long string on the PageHeader component, it should be show tooltip on hover when the title got truncated.

**Comment by hiteshk-dev** [comment]
@anuradha9712 please assign this to me

**Comment by anuradha9712** [comment]
@HiteshKumar000 Is there any update on this issue?

**Comment by hiteshk-dev** [comment]
Yes this is done on local will set a time to show it to you and then can
raise an MR for it
